draft-ietf-core-coap-tcp-tls.md

```diff
@@ -138,7 +138,8 @@ reliable delivery, simple congestion control, and flow control.
 Some environments benefit from the availability of CoAP carried over reliable
 transports such as TCP or TLS. This document outlines the changes required to use
 CoAP over TCP, TLS, and WebSockets transports. It also formally updates {{RFC7641}}
-for use with these transports.
+for use with these transports, and RFC 6455 to extend well-known
+sockets (RFC 5785) to the ws and wss URI schemes.

```

Comment:
1. _to extend well-known **sockets**_ ?

Did you mean well-known locations? 

2. _RFC 6455_ should be {{RFC5455}} ?
2. _(RFC 5785)_ should be {{RFC5785}} ?